reflex-dom-core/src/Reflex/Dom/Old.hs

```diff
@@ -112,6 +113,7 @@ type MonadWidgetConstraints t m =
   , MonadSample t (Performable m)
   , MonadReflexCreateTrigger t m
   , PostBuild t m
+  , MonadMountStatus t m
```

Comment:
I'm not sure this should go in here.  `MonadWidget` is a bit of an ad-hoc collection of stuff, which ideally I'd like to deprecate in favor of something that makes a bit more sense.  We might, e.g., write a class `Widget`, which is explicitly designed to be beginner-friendly, and then move from there.